docs/guides/template-locations.md

```diff
@@ -23,6 +23,31 @@ Timber::$locations = array(
 );
 ```
 
+You can also use namespaces in your locations too, just define it as the value next to a path, for example:
```

Comment:
I think I’d put this section in an own section with a separate heading and move the section just before `## Changing the default folder for Twig files`

```
## Register your own namespaces

…
```

Reply:
Addressed in https://github.com/timber/timber/pull/1791/commits/9abb875f985827b741fc50006408848253c02c06

Reply:
Oops, maybe I wasn’t clear enough about where to place it. I meant that this heading could be added as a heading for the whole section that you added to the docs 😅. Sorry!

I’d add it before this line: `
You can also use namespaces in your locations too, just define it as the value next to a path, for example:`

Reply:
Got it, thanks. Updated in https://github.com/timber/timber/pull/1791/commits/2f22f6ea44e75ffd0c0b728c4201a0d90f3eb898